src/components/link-preview/link-preview.pw.tsx

```diff
@@ -0,0 +1,258 @@
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest aria-label={CHARACTERS.STANDARD} />);
+
+    const previewElement = linkPreview(page);
+    await previewElement.focus();
+
+    await expect(previewElement).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(
+      <LinkPreviewComponentTest aria-label={CHARACTERS.STANDARD} />,
+      {
+        hooksConfig: { focusRedesignOptOut: true },
+      }
+    );
+
+    const previewElement = linkPreview(page);
+    await previewElement.focus();
+
+    await expect(previewElement).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 2px"
+    );
+  });
+});
+
+test.describe("check props for Link Preview component", () => {
+  (["div", "a"] as LinkPreviewProps["as"][]).forEach((as) => {
+    test(`should render Link Preview as prop using ${as}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest as={as} />);
+
+      await expect(linkPreviewAs(page, as)).toBeVisible();
+    });
+  });
+
+  testData.forEach((title) => {
+    test(`should render Link Preview title prop using ${title} as special character`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest title={title} />);
+
+      await expect(page.getByText(title)).toBeAttached();
+    });
+  });
+
+  testData.forEach((description) => {
+    test(`should render Link Preview description prop using ${description} as special character`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkPreviewComponentTest description={description} />);
+
+      await expect(page.getByText(description)).toBeAttached();
+    });
+  });
+
+  test("should render Link Preview with isLoading prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest isLoading />);
+
+    const previewComponent = page.locator(`[data-component="preview"]`);
+
+    const count = await previewComponent.count();
+
+    expect(count).toBe(4);
+  });
+
+  test("should render Link Preview with Image props", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <LinkPreviewComponentTest image={{ url: urlProp, alt: testCypress }} />
+    );
+
+    await expect(page.locator("img")).toHaveAttribute("src", urlProp);
+
+    await expect(page.locator("img")).toHaveAttribute("alt", testCypress);
+  });
+
+  test("should render Link Preview with url prop", async ({ mount, page }) => {
+    await mount(<LinkPreviewComponentTest url="https://www.foo.com" />);
+
+    await expect(page.getByText("www.foo.com")).toBeAttached();
+  });
+});
+
+test.describe("check functionality for Link Preview component", () => {
+  test("should verify hover color of Link Preview component", async ({
```

Comment:
**suggestion: (non-blocking)** might be nice to trim these tests descriptions a bit. Otherwise we'll have `Link Preview` mentioned three times in the testing block. In your version it'll end up being `Link Preview -> Check functionality of Link Preview component -> should verify hover color for Link Preview component` as an example. 

```suggestion
  test("should verify hover color", async ({
```